apps/ensadmin/src/components/registrar-actions/resolve-and-display-registrar-actions-panel.tsx

```diff
@@ -0,0 +1,31 @@
+import { useActiveNamespace } from "@/hooks/active/use-active-namespace";
+
+import { DisplayRegistrarActionsPanel } from "./display-registrar-actions-panel";
+import { useFetchRegistrarActions } from "./use-fetch-registrar-actions";
+
+interface ResolveAndDisplayRegistrarActionsPanelProps {
+  maxItems: number;
+
+  title: string;
+}
+
+/**
+ * Resolves Registrar Actions through ENSNode and displays the Registrar Actions Panel.
+ */
+export function ResolveAndDisplayRegistrarActionsPanel({
```

Comment:
```suggestion
/**
 * Fetches Registrar Actions through ENSNode and displays the Registrar Actions Panel.
 */
export function FetchAndDisplayRegistrarActionsPanel({
```
Goal: We should only use "resolve" terminology in the context of ENS resolution, not all API requests.


Edit: Please also update the name of this file.